skycoin/skywire
# Add Ping and Pong latency packets

Did you run `make format && make check`?
yes

Fixes #1261 

 Changes:	
- Renamed `NetworkProbePacket` to `PingPacket`
- Added `PongPacket`
- `PingPacket` now responds with `PongPacket`

How to test this PR:
1. Start `vpn-server` on visor A
2. From visor B connect to the `vpn-server` via the `vpn-client`
3. Check `Upload`, `Download` and `Latency` in the VPN UI
